defaults:
  - door_dbopt

bopt_agent:
  gmm:
    model: "models/gmm/door_em_f_3p.npy"
    type: force
    force_norm: 2.6178633634847966e-05
  base_accuracy: 0.34
